src/coreclr/build.cmd

```diff
@@ -277,6 +277,13 @@ REM Determine if this is a cross-arch build. Only do cross-arch build if we're a
 
 if %__SkipCrossArchNative% EQU 0 (
     if %__BuildNative% EQU 1 (
+        if %__BuildCrossArchNative% EQU 0 (
+            if /i not "%__BuildArch%"=="x86" (
+                REM Make recursive calls to build the linux & alpine cross OS DACs
+                call :BuildCrossOSDac -linuxdac
+                call :BuildCrossOSDac -alpinedac
```

Comment:
How does Linux and Alpine differ?

Reply:
It is probably nothing anymore.  Originally the offset of the `g_DACTable` differed.  Now that we are finding it from the runtime symbols that shouldn't matter anymore.

I will look at it.

Reply:
If we are building a single DAC, I would expect it to be the linux DAC. Is there a reason why you picked alpine? Is alpine going to work for non-alpine linux builds?

Reply:
Our primary customer is only requesting alpine builds.  At the moment, I am not aware of anything in the DAC which depends on Alpine vs. Linux.  However #32331 apparently broke Alpine debug, so there may be a difference.  If there is I will add Linux in CI builds.

Reply:
I guess since it is supposed to work on both, `-linuxdac` would be more natural. I was just worrying about if someone introduced a difference...  I have a slight bias to leaving it this way, but I see your reasoning.  Let me know which you prefer.

Reply:
I would prefer linuxdac